plugins/websocket_server/ign.js

```diff
@@ -181,17 +171,8 @@ Topic.prototype.subscribe = function(_callback) {
     // Register the callback with the topic name
     that.ign.on(that.name, _cb);
 
-    // \todo: Check that requesttMsgType is valid
-    var requestMsgType = that.ign.root.lookup("ignition.msgs.WebRequest");
-
-    // Create the subscription request message
-    var webRequestMsg = requestMsgType.encode({
-      operation: "subscribe",
-      topic: that.name,
-    }).finish();
-
     // Send the subscription message over the websocket.
-    that.ign.sendMsg(webRequestMsg);
+    that.ign.sendMsg(buildMsg(['sub', that.name, '', '']));
```

Comment:
Can't comment there, but on line 183, `emiiter(_callback)` → `emitter(_callback)` (`else` branch on the `if (!this.ign.isConnected)` block)

Reply:
Thanks, fixed in 87d5a5164086c65979e3f1ce39c873c03ea99361